### DISCUSSION TITLE
Run Norfair On Yolov5 On Image Data

### DISCUSSION THREAD

**Discussion by MuhammadBilal848** [tryolabs/norfair#273] [General] [answered]
Hello! I want to run norfair on my **pretrained yolov5 model**. I have seen the norfair **implementation of yolov5** but its for video inference idk how to run it on images.
Can someone help?

**Comment by facundo-lezama** [comment, marked as answer]
Hi @MuhammadBilal848! You can replace the `for frame in video` statement with something like `for frame in frames` where `frames` contains the list of images you want to run Norfair on.
